tests/driver_itg320x/README.md

```diff
@@ -0,0 +1,32 @@
+# About
+
+This is a manual test application for the InvenSense ITG320X 3-axis gyroscope driver.
+
+# Usage
+
+The test application demonstrates the use of the ITG320X. It uses the
+default configuration parameters:
+
+- Low pass filter bandwidth (LPF_BW) of 5 Hz (```ITG320X_LPF_BW_5```)
+- Internal sample rate (ISR) of 1 kHz which results from LPF_BW of 5 Hz
+- Internal sample rate divider (ISR_DIV) of 9 (```ITG320X_PARAM_ISR_DIV```)
+- Output data rate (ODR) of 100 Hz resulting from
+     ODR = ISR / (ISR_DIV + 1)
+
+The application may use two different approaches to retrieve new data,
+either
+
+- periodically fetching the data at a rate lower than the sensor's
+  output data rate (ODR), or
+- fetching the data when the data-ready interrupt is triggered.
+
+To use the latter approach, the according GPIO to which the sensor's
+**INT** output pin is connected has to be configured and initialized
+by the application. In this test application, this is done by defining
+```USE_ITG320X_DRDY``` and overrding the default configuration
```

Comment:
It's `USEMODULE += itg320x_int` now.

Reply:
Ah yeah, I changed it in `main.c` and forgot it in `README.md` :sunglasses: